package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "quill-cursors",
-  "version": "3.1.2",
+  "version": "3.2.0",
```

Comment:
Let's be conservative and bump a major version, since we're changing (including removing) some CSS classes, which people might be overriding, etc.

Also please update the [`CHANGELOG`](https://github.com/reedsy/quill-cursors/blob/main/CHANGELOG.md) with the breaking changes.